benchmark/bench-show/bench-chart.cabal

```diff
@@ -0,0 +1,20 @@
+cabal-version:      2.2
+name:               bench-chart
+version:            0.0.0
+synopsis:           Chart for Benchmarks
+description: Chart is separated from the benchmarks.
```

Comment:
We can say something like - "Benchmark reporting application is not included in the overall cabal project so that the dependencies of both can remain independent. Benchmark reporting has a lot of dependencies that usually lag behind when new GHC releases arrive."